nav2_bt_navigator/behavior_trees/navigate_w_replanning_and_recovery.xml

```diff
@@ -20,12 +20,13 @@
       </PipelineSequence>
       <ReactiveFallback name="RecoveryFallback">
         <GoalUpdated/>
-        <SequenceStar name="RecoveryActions">
+        <RoundRobin name="RecoveryActions">
           <ClearEntireCostmap name="ClearLocalCostmap-Subtree" service_name="local_costmap/clear_entirely_local_costmap"/>
           <ClearEntireCostmap name="ClearGlobalCostmap-Subtree" service_name="global_costmap/clear_entirely_global_costmap"/>
           <Spin spin_dist="1.57"/>
           <Wait wait_duration="5"/>
-        </SequenceStar>
+          <BackUp backup_dist="-.15" backup_speed=".025" />
```

Comment:
```suggestion
          <BackUp backup_dist="-0.15" backup_speed=".025" />
```

Not that I mind much it being added, but why?

Reply:
BackUp was added at request of Sarthak , @naiveHobo